tensorflow/lite/tools/accuracy/ilsvrc/imagenet_model_evaluator.cc

```diff
@@ -141,6 +141,8 @@ class CompositeObserver : public ImagenetModelEvaluator::Observer {
       tflite::Flag::CreateFlag(kNumRanksFlag, &params.num_ranks,
                                "Generates the top-1 to top-k accuracy values"
                                "where k = num_ranks. Default: 10"),
+      tflite::Flag::CreateFlag("nnapi_allow_fp16", &params.nnapi_allow_fp16,
+                               "allow fp16 in nnapi"),
```

Comment:
I think we don't need to change this code as "tools/evaluation/tasks/imagenet_image_classification" is a better version of this one. We should have done some more proactive legacy code clean up.

Reply:
I like the output of this one (top-1, .., top-10 for each image)
```
...
67.742, 83.871, 87.097, 90.323, 90.323, 90.323, 93.548, 93.548, 93.548, 93.548
68.750, 84.375, 87.500, 90.625, 90.625, 90.625, 93.750, 93.750, 93.750, 93.750
69.697, 84.848, 87.879, 90.909, 90.909, 90.909, 93.939, 93.939, 93.939, 93.939
70.588, 85.294, 88.235, 91.176, 91.176, 91.176, 94.118, 94.118, 94.118, 94.118
...
```

Reply:
Acked. I checked the code again, and found this tool actually supports the delegate registrar mechanism, i.e. the new option you added will be automatically supported w/ this tool :-)

Reply:
Yes, it works. I tried. Thanks